Distribution/Server/Users/AuthToken.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE DeriveDataTypeable, GeneralizedNewtypeDeriving, TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies #-}
+module Distribution.Server.Users.AuthToken
```

Comment:
Yeah, some consistently used name would be nice. Is this an auth token or an API key or what? :-)